CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # `chainweb-node` Changelog
 
+## 2.17.1 (2022-12-02)
+
+This is a feature and bug-fix release. Upgrading is optional but recommended.
+
+To upgrade, pull the latest docker image or download the binary and restart the
+node.
+
+All 2.17* versions expire on **2023-03-02T00:00:00Z**.
+
+[Changes](https://github.com/kadena-io/chainweb-node/compare/2.17...2.17.1):
+
+*   Add transaction simulator to cwtools. (#1558)
```

Comment:
Since this isn't in prod code path do we want to put this in toplevel "Changes"?

Reply:
I thought it would be nice to have something else beside just bug fixes. Also, it's a nice new feature. But, yeah, it is not relevant for most users.